### PR TITLE
Update random_tf_environment.py

### PR DESCRIPTION
**Pull request by ngroves08**
Includes support for array based reward specs

### CODE REVIEW

**Comment by ngroves08** [comment]
@googlebot I signed it!

File: tf_agents/environments/random_tf_environment.py [hunk 1]
```diff
@@ -88,7 +88,7 @@ def _sample_obs_and_reward(self):
   def _reset(self):
     """Resets the environment and returns the current time_step."""
     obs, _ = self._sample_obs_and_reward()
-    time_step = ts.restart(obs, self._batch_size)
+    time_step = ts.restart(obs, self._batch_size, reward_spec = self._time_step_spec.reward)
```
Comment (ebrevdo) [inline tf_agents/environments/random_tf_environment.py]: nit: can you remove the spaces around the equals sign?

Comment (ngroves08) [reply tf_agents/environments/random_tf_environment.py]: Updated code in my version - hopefully this flows through to the PR

**Comment by tfboyd** [comment]
@ebrevdo   Is this PR ok to +2 and get and LGTM?

**Review by ebrevdo — APPROVED** [review]

**Pull request merged**